packages/wallet/src/PersonalWallet/PersonalWallet.ts

```diff
@@ -225,6 +225,7 @@ export class PersonalWallet implements ObservableWallet {
   readonly handleProvider: HandleProvider;
   readonly changeAddressResolver: ChangeAddressResolver;
   readonly publicStakeKeys$: TrackerSubject<PubStakeKeyAndStatus[]>;
+  drepPubKey: Ed25519PublicKeyHex;
```

Comment:
let's make this private to avoid having it modified by external means.

Reply:
done in [274e020](https://github.com/input-output-hk/cardano-js-sdk/pull/936/commits/274e0201bc9165a19d955ff31a8c41e6bebc0f59)